src/DB/entities/index.ts

```diff
@@ -0,0 +1,3 @@
+const entities = [];
+
+export default entities;
```

Comment:
po co to jest?

Reply:
dodałem testowe entity i nie śmigało gdy miałem
```
entities: [
            __dirname + '/../**/*.entity{.ts,.js}',
        ]
```
a dodając poszczególne entity do tej tablicy wszystko działa.

Reply:
usunąłem - udało mi się ostatecznie podając odpowiednią ścieżkę dla entities